### DISCUSSION TITLE
Error on yq eval with env vars

### DISCUSSION THREAD

**Discussion by adusumillipraveen** [mikefarah/yq#654] [Q&A] [answered]
Trying to upgrade to latest 4.x and unable to make eval commands work with env vars. Raising it here as i wasn't sure if I am doing anything silly ( below commands is taken from examples in https://mikefarah.gitbook.io/yq/v/v4.x/env-variable-operators)

Command :

`yq eval --null-input '.a = env(myenv)'`

Error:

```
Error: Parsing expression: Lexer error: could not match text starting at 1:6 failing at 1:8.
	unmatched text: "en"
```

**Comment by mikefarah** [comment, marked as answer]
Make sure you are using 4.3.1 (I accidently released 4.3.0 without merging in the env branch 🤦 )

**Comment by adusumillipraveen** [comment]
Just realised homebrew installs yq 4.2.1 as latest. Is this a bug or something ?

**Reply by mikefarah** [reply]
Seems to be up to date now: https://formulae.brew.sh/formula/yq

**Reply by adusumillipraveen** [reply]
Thanks, i got it working now.
